StudyHub/StudyHub/Controllers/AssignmentController.cs

```diff
@@ -1,11 +1,13 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
 using StudyHub.BLL.Services.Interfaces;
 using StudyHub.Common.DTO.Assignment;
 
 namespace StudyHub.Controllers;
 
 [Route("api/[controller]")]
 [ApiController]
+[Authorize]
```

Comment:
check if we can put here Autorize[Roles = "Teacher"] and override specific endpoint with another rule

Reply:
no, i can't do it